guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -136,10 +136,8 @@ ifdef::katello,satellite,orcharhino[]
 . Optional: Select the *Ignore errors* option if you want to ignore subscription manager errors.
 . Optional: Select the *Force* option if you want to remove any `katello-ca-consumer` rpms before registration and run `subscription-manager` with the `--force` argument.
 endif::[]
-
-ifdef::foreman-el,foreman-deb,katello[]
-. Optional: This step is for the Katello users only.
-If you register {RHEL} or CentOS hosts, in the *Activation Keys* field, enter one or more activation keys to assign to registered hosts.
+ifdef::katello[]
+If you register {EL} hosts, in the *Activation Keys* field, enter one or more activation keys to assign to registered hosts.
```

Comment:
```suggestion
If you register {EL} hosts, in the *Activation Keys* field, enter one or more activation keys to assign to the registered hosts.
```

Reply:
alternative: "to your registered hosts"?

Reply:
Technically it was already there before and I think in this case not having an article is also correct, but I'll note I'm not a native speaker.

Reply:
Do as you see fit, but I am fairly certain an article or a possesive pronoun belongs there. Anyway, it is a non-blocking suggestion.